navmodel-samples/src/main/kotlin/com/bumble/appyx/navmodel/tiles/transitionhandler/TilesTransitionHandler.kt

```diff
@@ -62,7 +63,7 @@ class TilesTransitionHandler<T>(
 
 @Composable
 fun <T> rememberTilesTransitionHandler(
-    transitionSpec: TransitionSpec<Tiles.State, Float> = { spring() }
+    transitionSpec: TransitionSpec<Tiles.State, Float> = { spring(stiffness = Spring.StiffnessVeryLow) }
```

Comment:
is this related to the PR?

Reply:
Not strictly. I changed it while testing, as the default spring makes it very difficult to see what's happening.